example/chessground/lib/options_screen.dart

```diff
@@ -0,0 +1,202 @@
+import 'dart:async';
+
+import 'package:ble_chess_peripheral_driver/chess_peripheral_driver.dart';
+import 'package:flutter/material.dart';
+
+class OptionsScreen extends StatefulWidget {
+  const OptionsScreen({required this.peripheral, super.key});
+
+  final Peripheral peripheral;
+
+  @override
+  State<OptionsScreen> createState() => OptionsScreenState();
+}
+
+class OptionsScreenState extends State<OptionsScreen> {
+  StreamSubscription? _subscription;
+
+  Peripheral get peripheral => widget.peripheral;
+  bool get areOptionsInitialized => peripheral.areOptionsInitialized;
+  List<Option> get options => peripheral.options;
+
+  @override
+  void initState() {
+    super.initState();
+    _subscription = peripheral.optionsUpdateStream.listen(_updateOptions);
+    peripheral.handleOptionsBegin();
+  }
+
+  @override
+  void dispose() {
+    _subscription?.cancel();
+    super.dispose();
+  }
+
+  void _updateOptions(_) {
+    setState(() {});
+  }
+
+  String _convertToReadable(String str) => str
```

Comment:
str -> option

Reply:
It uses all options names and for enum options values